rules/config

```diff
@@ -56,5 +56,11 @@ SONIC_ROUTING_STACK = quagga
 # Enable Origanization Extensions - Specific to the deployment scenarios of the Organization
 ENABLE_ORGANIZATION_EXTENSIONS = y
 
+# Debugging option allows sonic debian packages to get built including symbols
+# information. Profiling option, disables compiler optimizations (-O0) as well
+# as includes symbols information too.
+#SONIC_DEBUGGING_ON = y
```

Comment:
I feel as though this is a bit confusing, as we also have the `SONIC_CONFIG_DEBUG` option at line 43, which installs packages needed for debug.

Reply:
What about adding a better a description for SONIC_CONFIG_DEBUG option too? i can do something like: s/install debug packages/Install linux debugging tools/

Reply:
Or we can also change SONIC_DEBUGGING_ON to something else. Let me know if have a better  name for this.

Reply:
Maybe change `SONIC_CONFIG_DEBUG` to something like `SONIC_CONFIG_DEBUG_TOOLS`, `SONIC_CONFIG_INSTALL_DEBUG_TOOLS` or `SONIC_INSTALL_DEBUG_TOOLS`?

Reply:
Will go ahead with this one: SONIC_INSTALL_DEBUG_TOOLS